wiki/DEBUGGING.md

```diff
@@ -0,0 +1,128 @@
+# An Introduction to Debugging in Haskell
+
+## Introduction
+
+There is a general purpose debugging guide available [here](https://wiki.haskell.org/Debugging), but it
```

Comment:
In general, do not worry about line length. Instead, write each *sentence* on a single line. This will make diffs more useful when we make future updates.